packages/overlay/src/Overlay.ts

```diff
@@ -516,6 +516,13 @@ export class Overlay extends ComputedOverlayBase {
             return;
         }
 
+        // Earlier this nextFrame() was called in ensureOnDom. Without waiting here, the longpress state will be `null` when the overlay is opened and the overlay will close immediately.
```

Comment:
I appreciate the comment here explaing the change and preventing us from making untended regressions.